thirdparty/github.go

```diff
@@ -157,7 +158,7 @@ func GetGitHubMergeBaseRevision(oauthToken, repoOwner, repo, baseRevision string
 		repoOwner,
 		baseRevision,
 		repoOwner,
-		currentCommit.SHA)
+		*currentCommit.SHA)
```

Comment:
isn't it a better API to pass/return the sha as a string, rather than this weird pointer?